### PR TITLE
Common updates

### PR DESCRIPTION
**Pull request by JohnnyRealImpact**

### CODE REVIEW

File: db/migrations/0.1.0.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* globals db: true */
+
+/**
+ * @file MongoDB migration script for version 1.1.0.
```
Comment (Pyo25) [inline db/migrations/0.1.0.js]: Do we need stuffs in the `migrations` folder?

File: config/config.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "closeTimeout": 10000
+  },
+  "jwt": {
+    "publicKey": "config/jwt/public_key.pem"
+  },
+  "logger": {
+    "name": "snd-task-service"
+  },
+  "router": {
+    "allVersions": ["1.0.0", "1.1.0"]
```
Comment (Pyo25) [inline config/config.json]: Shouldn't it be `["0.1.0"]` ?

File: src/config/mock.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    retryTimeout: 1,
+    user: 'foo',
+    uri: 'foo'
+  };
+
+  config.router = {
+    host: 'router-host',
+    port: 0,
+    protocol: 'router-protocol',
+    allVersions: ['1.0.0', '1.1.0']
```
Comment (Pyo25) [inline src/config/mock.js]: `0.1.0` ?

File: src/routes/status.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+var Server = require('../server');
+
+/**
+ * @class
+ * @classdesc Controller for the status routes.
+ * @param {module:logger} logger The logger instance.
+ * @param {module:server} server The server instance.
+ */
+function StatusController(logger, server) {
+  var version = '1.0.0';
```
Comment (Pyo25) [inline src/routes/status.js]: This should come from the `package.json` right?

File: config/config.json (hunk 2 above)
Comment (JohnnyRealImpact) [inline config/config.json]: You are absolutely right.... :)
https://github.com/RealImpactAnalytics/snd-task-service/pull/1/commits/4e372b682c99af8e5f00e991731ce9704668bebf

File: src/config/mock.js (hunk 3 above)
Comment (JohnnyRealImpact) [inline src/config/mock.js]: https://github.com/RealImpactAnalytics/snd-task-service/pull/1/commits/4e372b682c99af8e5f00e991731ce9704668bebf

File: db/migrations/0.1.0.js (hunk 1 above)
Comment (JohnnyRealImpact) [inline db/migrations/0.1.0.js]: https://github.com/RealImpactAnalytics/snd-task-service/pull/1/commits/1c0243265122178f7d012b5e6e47c94a79eac993

File: src/routes/status.js (hunk 4 above)
Comment (JohnnyRealImpact) [inline src/routes/status.js]: Indeed: https://github.com/RealImpactAnalytics/snd-task-service/pull/1/commits/a0890f0cdc313299e9e12cea36911d212e9b0182

**Comment by JohnnyRealImpact** [comment]
@Pyo25 - I have updated the code according to your observations. 
Should be ready for merge now. :)

**Pull request merged**